litho-testing/src/main/java/com/facebook/litho/testing/TestDrawableComponent.java

```diff
@@ -30,59 +30,32 @@
   private static final Pools.SynchronizedPool<Builder> mBuilderPool =
       new Pools.SynchronizedPool<>(2);
 
-  private final boolean mCallsShouldUpdateOnMount;
-  private final boolean mIsPureRender;
-  private final boolean mCanMeasure;
-  private final boolean mUsesDisplayList;
-  private final boolean mImplementsAccessibility;
-  private final boolean mIsMountSizeDependent;
-
-  private synchronized static TestDrawableComponent get(
-      boolean callsShouldUpdateOnMount,
-      boolean isPureRender,
-      boolean canMeasure,
-      boolean implementsAccessibility,
-      boolean usesDisplayList,
-      boolean isMountSizeDependent) {
+  private static final byte CALLS_SHOULD_UPDATE_ON_MOUNT = 1 << 0;
+  private static final byte IS_PURE_RENDER = 1 << 1;
+  private static final byte CAN_MEASURE = 1 << 2;
+  private static final byte USES_DISPLAY_LIST = 1 << 3;
+  private static final byte IMPLEMENTS_ACCESSIBILITY = 1 << 4;
+  private static final byte IS_MOUNT_SIZE_DEPENDENT = 1 << 5;
+
+  private byte mProperties;
```

Comment:
`byte` allows us to keep only 8 flags, but we already have 6 values, so let's use `long` to have enough space for future flags. This will involve changing the flag themselves above as well.